bin/impl/redeploy.sh

```diff
@@ -35,21 +35,20 @@ elif [ -n "$FLUO_TARBALL_REPO" ]; then
   cd $FLUO_TARBALL_REPO
   mvn clean package -DskipTests -Daccumulo.version=$ACCUMULO_VERSION -Dhadoop.version=$HADOOP_VERSION -Dthrift.version=$THRIFT_VERSION
 
-  TARBALL=$FLUO_TARBALL_REPO/modules/distribution/target/fluo-$FLUO_VERSION-bin.tar.gz
+  TARBALL=$FLUO_TARBALL_REPO/modules/distribution/target/$FLUO_TARBALL
   if [ ! -f $TARBALL ]; then
     echo "The tarball $TARBALL does not exist after building from the FLUO_TARBALL_REPO=$FLUO_TARBALL_REPO"
     echo "Does your repo contain code matching the FLUO_VERSION=$FLUO_VERSION set in env.sh?"
     exit 1
   fi
-elif [ -n "$FLUO_TARBALL_URL" ]; then
-  TARBALL_FILENAME=fluo-distribution-$FLUO_VERSION-bin.tar.gz
-  if [ -f "$DOWNLOADS/$TARBALL_FILENAME" ]; then
-    echo "$TARBALL_FILENAME already exists in downloads/"
+elif [ -n "$FLUO_TARBALL_URL_PREFIX" ]; then
+  if [ -f "$DOWNLOADS/$FLUO_TARBALL" ]; then
```

Comment:
Should redeploy.sh download?    Thinking it would be more consistent if only download.sh did that.